finetune/base_models/bert/model.py

```diff
@@ -99,6 +106,32 @@ def get_optimal_params(cls, config):
                 "batch_size": base_batch_size,
                 "chunk_context": 16,
                 "predict_batch_size": 256 if config.float_16_predict else 48,
+                "mixed_precision": False,
+                "float_16_predict": False,
+                "lr": base_learning_rate,
+            }
+        elif config.optimize_for.lower() == "accuracy_fp16":
+            overrides = {
+                "max_length": base_max_length,
+                "n_epochs": base_n_epochs,
+                "batch_size": 12,
+                "chunk_context": None,
+                "predict_batch_size": 40,
+                "mixed_precision": True,
+                "float_16_predict": True,
+                "lr": base_learning_rate * 6,
```

Comment:
How is lr rescaling calculated?  Is it something we could auto-apply based on whether or not fp16 is used rather than requiring manual configuration w/ the optimize_for defaults aren't used?  I could see this being a common issue with configuration

Reply:
As above. Probably yes, but it would come at added complexity to the config resolution. For example if we set float. Personally think if you are departing from the presets there should be as little magic as possible.

Reply:
I'm happy with that decision.